db/migrate/20181024163850_add_source_type.rb

```diff
@@ -0,0 +1,15 @@
+class AddSourceType < ActiveRecord::Migration[5.1]
+  def change
+    create_table :source_types, :id => :bigserial do |t|
+      t.string :name
+      t.string :display_name
+      t.string :vendor
```

Comment:
For Vendor I was thinking `Red Hat`, `Microsoft`, etc... so that we split the "company" from the "product".  This is a bit confusing in MIQ because we aren't consistent in what is the vendor vs what is the product: Microsoft::InfraManager => SCVMM by Microsoft and Azure::CloudManager => Azure by Microsoft, Redhat::InfraManager => RHV by Red Hat and Openshift::ContainerManager => Openshift by Red Hat